build.sbt

```diff
@@ -61,4 +63,4 @@ trait MicroService {
       )
     )
     .enablePlugins(SbtDistributablesPlugin, SbtAutoBuildPlugin, SbtGitVersioning)
-}
+
```

Comment:
Need to add entry for  .disablePlugins(JUnitXmlReportPlugin) to sbt file otherwise the build will fail

Reply:
Done